### DISCUSSION TITLE
strongSwan 6 release date now that NIST has published final PQC specs?

### DISCUSSION THREAD

**Discussion by stevedoyle** [strongswan/strongswan#2412] [Q&A]
What is the timeline for a strongSwan 6.0 release?

This question was previously asked in #1850, where the answer indicated that the lack of published PQC algorithm standards was one item blocking a 6.0 release. Now that NIST has published the final specifications for ML-KEM, ML-DSA and SH-DSA, is there a target timeline for a strongSwan 6.0 release?

**Comment by strongX509** [comment]
We are waiting for IANA to assign the IKEv2 Key Exchange codepoints for the ML-KEM algorithms. Also according to the liboqs forum the final ML-KEM source code isn't available yet since there were some last minute changes relative to the draft standard.

**Reply by CyrilStormshield** [reply]
Hi, is there any change about this topic given IANA has assigned the code points for ML-KEM (https://www.iana.org/assignments/ikev2-parameters/ikev2-parameters.xhtml#ikev2-parameters-8) ?
